src/lib/redux-store/utils/handleErrorResponse.ts

```diff
@@ -0,0 +1,10 @@
+import type { ActionResponse } from "$common/data-modeler-service/response/ActionResponse";
+import { ActionStatus } from "$common/data-modeler-service/response/ActionResponse";
+import notifications from "$lib/components/notifications";
+
+export function handleErrorResponse(actionResponse: ActionResponse) {
+  if (!actionResponse || actionResponse.status === ActionStatus.Success) return;
+
+  const actionResponseMessage = actionResponse.messages[0];
+  notifications.send({ message: actionResponseMessage.message });
```

Comment:
This is nice, but I think the better approach from a UX point of view would be to gray out the rest of the UX. I think this will take a bit of UX work to refactor to make it work out. Is there a status for the metrics definition that we could surface here? I would like to get rid of the quick start button and say that the model doesn't exist in its place.